docs/index.md

```diff
@@ -56,9 +56,9 @@ provider "tanzu-mission-control" {
 
 ### Required
 
-- **endpoint** (String)
-- **vmw_cloud_api_token** (String, Sensitive)
+- `endpoint` (String)
```

Comment:
Why is this change needed ?

Reply:
That was taken in as we hadn't run the documentation, the change was as per some old version. I have reverted it.